README.md

```diff
@@ -347,3 +347,54 @@ let result = client
 
 Consumers do not need to specify parameters and struct properties that are not
 required or for which the API specifies defaults. Neat!
+
+#### Enabling the builder style in build.rs
+
+To enable the builder style, the `build.rs` file should look something like this:
+
+```rust
+fn main() {
+    let src = "../sample_openapi/keeper.json";
+    println!("cargo:rerun-if-changed={}", src);
+    let file = std::fs::File::open(src).unwrap();
+    let spec = serde_json::from_reader(file).unwrap();
+    let mut binding = GenerationSettings::default();
+    let settings = binding.with_interface(InterfaceStyle::Builder);
+    let mut generator = progenitor::Generator::new(&settings);
+    let tokens = generator.generate_tokens(&spec).unwrap();
+    let ast = syn::parse2(tokens).unwrap();
+    let content = prettyplease::unparse(&ast);
+
+    let mut out_file = std::path::Path::new(&std::env::var("OUT_DIR").unwrap()).to_path_buf();
+    out_file.push("codegen.rs");
+
+    std::fs::write(out_file, content).unwrap();
+}
+```
+
+## Changing default client settings
+
+Currently, the generated code doesn't deal with request headers. To add default headers to all requests, you can use the default_headers method when constructing the Client.
+
+```rust
+    let baseurl = "https://company.com/api/v2";
+    
+    let access_token = "OP7nhQY46jf3I2sqyTZBHWEOIRsldfWEFlkjRkrjlER";
```

Comment:
would it make sense to have an example that reads an environment variable to encourage people not to accidentally leak credentials?